CHANGELOG

```diff
@@ -1,22 +1,30 @@
 snowplow-web 0.15.0 (2023-0X-XX)
 ---------------------------------------
 ## Summary
-This version contains moves the iab, ua, and yauaa contexts into the base events this run table for Postgres/Redshift, completing the work to decouple our sessions models from the page view models.
+The main change in this version is the addition of many new fields to the `sessions` table, including the ability to define your conversion events that are aggregated per session. You can read more about it in our docs [here](https://docs.snowplow.io/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-web-data-model/conversions/)! We also add the (optional) count of events per event type for each session, you can enable it by turning on the `snowplow__list_event_counts` variable.
+
+This version also moves the iab, ua, and yauaa contexts into the base events this run table for Postgres/Redshift, completing the work to decouple our sessions models from the page view models for all warehouses.
+
+## 🚨 Breaking Changes 🚨
+While not breaking in that your models should all still run, there are many new columns in the sessions and users tables that will be automatically added to your derived tables, which may have unintended consequences with existing queries or BI tools that use column position (although dbt usually appends these columns to the end).
```

Comment:
Does this then also trigger a full refresh sync to backfill those columns or does dbt just leave them blank?

Reply:
It leaves them blank and fills them onwards, if they want them filled they will need to do a full refresh https://docs.getdbt.com/docs/build/incremental-models#what-if-the-columns-of-my-incremental-model-change